docs/styleguides/python.md

```diff
@@ -0,0 +1,173 @@
+# DCOS Python Style Guide
+###### Version 0.1
```

Comment:
Yeah, better to call out version. We can feel free to update / enhance / clarify the style guide at the given version. But if we make any rule changes, than a version update will be desirable.